draft-ietf-oauth-identity-chaining.md

```diff
@@ -53,7 +53,7 @@ informative:
 
   I-D.ietf-oauth-security-topics:
   I-D.ietf-oauth-resource-metadata:
-
+  I-D.draft-ietf-oauth-v2-1-12:
```

Comment:
I think you can and probably should leave the draft # off and the tooling will do more or less what you'd want 
```suggestion
  I-D.draft-ietf-oauth-v2-1:
```